config/routes.rb

```diff
@@ -28,6 +28,10 @@
         end
       end
       get 'create_key' => 'products#create_key'
+      get 'new_from_git_remote' => 'products#new_from_git_remote'
+      get 'new_from_rails_template' => 'products#new_from_rails_template'
+      post 'create_from_git_remote' => 'products#create_from_git_remote'
+      post 'create_from_rails_template' => 'products#create_from_rails_template'
```

Comment:
"Doing this changes the routes"

What do the routes look like without this change? i.e. can you show the relavent output of `rake routes` for both?